client/css/style.css

```diff
@@ -1671,6 +1671,36 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	margin-top: 0.2em;
 }
 
+.password-container {
+	position: relative;
+}
+
+#sign-in .password-container {
+	width: 100%;
+}
+
+.password-container .see-pw {
```

Comment:
Add `cursor: pointer` and `:hover` styles to change color to something more visible.